WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -16585,6 +16595,186 @@
 			shellScript = "$SRCROOT/../Scripts/BuildPhases/GenerateCredentials.sh\n";
 			showEnvVarsInLog = 0;
 		};
+		09DBEA4C281329ED0019724E /* [Lint] Check AppLocalizedString usage */ = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "[Lint] Check AppLocalizedString usage";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = "/bin/bash -eu";
+			shellScript = "\"$SRCROOT/../Scripts/BuildPhases/runRubyScript\" \"LintAppLocalizedStringsUsage.rb\" \"${TARGET_NAME}\"\n";
+			showEnvVarsInLog = 0;
+		};
+		09DBEA4D281333060019724E /* [Lint] Check AppLocalizedString usage */ = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "[Lint] Check AppLocalizedString usage";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = "/bin/bash -eu";
+			shellScript = "\"$SRCROOT/../Scripts/BuildPhases/runRubyScript\" \"LintAppLocalizedStringsUsage.rb\" \"${TARGET_NAME}\"\n";
```

Comment:
This is the new way I used to wrap the call to the script into proper ceremony to setup `rbenv`/`rvm`/`bundle exec` for its execution.

I could have just made it a `RunLintAppLocalizedStringsUsage.sh` script specific for that one, which would have the call to `bundle exec ruby LintAppLocalizedStringUsage.rb "$TARGET_NAME"` hardcoded in it, but I figured it could be useful to have something more generic like `runRubyScript` which could wrap any arbitrary ruby script, so that if in the future we need to add other Build Phases based on ruby scripts, this could be reused (and also, if that works well, I might steal it for some other personal OSS projects that might also use ruby scripts for script build phases… 😇 )

Reply:
+1. I like that this is generic 👍